src/Parallel-Tests.ps1

```diff
@@ -23,7 +23,14 @@ foreach ($a in $assemblies)
 {
     $xmlName = 'xUnit-Results-' + [System.IO.Path]::GetFileNameWithoutExtension($a) + '.xml'
     $outXml = $(Join-Path $outDir $xmlName)
-    $cmdLine = $xunitRunner.FullName + ' ' + $a + ' ' + $testFilter + ' -xml ' + $outXml + ' -parallel none -noshadow -verbose' 
+	if([System.IO.Path]::GetFileNameWithoutExtension($a) -eq 'DefaultCluster.Tests')
```

Comment:
actually it does not have real time gain... isn't it supposed to be faster? am I doing it wrong?